### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AyoubM490**

### CODE REVIEW

**Comment by AyoubM490** [comment]
I think your README.md file looks good

File: index.html [hunk 1]
```diff
@@ -88,7 +88,7 @@ <h3>
       <section class="main-section" id="What_you_should_already_know">
         <h2>What you should already know</h2>
         <article>
-          <h3>This guide assumes you have the following basic background:</h3>
```
Comment (AyoubM490) [inline index.html:91]: I think you are missing an n there for background

File: style.css [hunk 2]
```diff
@@ -47,7 +47,7 @@ header {
   position: absolute;
   margin-left: 310px;
   padding: 20px;
-  margin-bottom: 110px;
```
Comment (AyoubM490) [inline style.css:50]: You should increase your margin-bottom by 10px.

**Review by alexander16108 — APPROVED** [review]
Hi @AyoubMs , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**